src/process.cpp

```diff
@@ -49,14 +49,6 @@ void process_end(bp::child &proc, bp::group &proc_handle) {
   proc.wait();
 }
 
-int exe_with_full_privs(const std::string &cmd, bp::environment &env, file_t &file, std::error_code &ec) {
```

Comment:
This is a great idea, but honestly this should be an entirely different pull request. Since it is more of a security concern in general, it's not ideal to mix multiple features or changes in one pull request generally.

Reply:
Yeah, in hindsight, it probably should have been a separate PR. I'll let @ReenigneArcher make the call on it. I'm happy to split the execution changes into a separate PR if you want.

Reply:
Well, as long as he agrees with it, #998 is the issue you can link for that PR

Reply:
I agree that this change should be a separate PR.